policy/handler.go

```diff
@@ -145,7 +147,14 @@ func (h *Handler) Run(ctx context.Context, evalCh chan<- *sdk.ScalingEvaluation)
 				}
 				h.log.Error(errors[0], "errors", errors[1:])
 			} else {
-				h.log.Error("encountered an error monitoring policy", "error", err)
+				if errHelper.APIErrIs(err, http.StatusNotFound, "not found") {
```

Comment:
I both cases, you log the error, there is m handling done. Is it worth it to make the code more complex for one extra line?

Reply:
this path is expected behavior, so Debug level. I figure in case the cleanup referred to in the comment doesn't happen as it's supposed to, this is information that might be nice to have.

the other path is an unexpected error, so logged as Error, which people may configure their monitoring system to alert them about.